twistr/game.go

```diff
@@ -153,14 +156,92 @@ func SelectRandomCard(s *State, player Aff) Card {
 	return s.Hands[player].Cards[n]
 }
 
+func actionsThisTurn(s *State, player Aff) int {
+	_, active := s.TurnEvents[NorthSeaOil]
+	switch {
+	case player == USA && active:
+		return 8
+	case s.SREvents[ExtraAR] == player:
+		return 8
+	default:
+		return s.ActionsPerTurn()
+	}
+}
+
+func outOfCards(s *State, player Aff) bool {
+	switch {
+	case s.ChinaCardPlayer == player && s.ChinaCardFaceUp:
+		return false
+	case len(s.Hands[player].Cards) > 0:
+		return false
+	default:
+		return true
+	}
+}
+
 func Turn(s *State) {
-	// Stub: awaiting implementation in issue#13
-	MessageBoth(s, fmt.Sprintf("TURN %d", s.Turn))
+	MessageBoth(s, fmt.Sprintf("== Turn %d", s.Turn))
+	if s.Turn > 1 {
```

Comment:
i think it's just that you get your first hand in initial setup, right?
i don't see a way to change this that's much better. open to ideas!